test-suite/src/test/groovy/views/ModelAndViewSpec.groovy

```diff
@@ -0,0 +1,137 @@
+/*
+ * Copyright 2017-2021 original authors
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ * http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+package views
+
+import io.micronaut.context.ApplicationContext
+import io.micronaut.http.HttpRequest
+import io.micronaut.http.HttpResponse
+import io.micronaut.http.HttpStatus
+import io.micronaut.http.client.HttpClient
+import io.micronaut.http.client.exceptions.HttpClientResponseException
+import io.micronaut.runtime.server.EmbeddedServer
+import io.micronaut.views.model.FruitsController
+import spock.lang.Specification
+
+class ModelAndViewSpec extends Specification {
+
+    def "a view model can be any object"() {
+        given:
+        EmbeddedServer embeddedServer = ApplicationContext.run(EmbeddedServer, [
```

Comment:
We can avoid repetition by using @Shared with `EmbeddedServer` creation.

Reply:
Doen via https://github.com/micronaut-projects/micronaut-views/pull/246/commits/a9ba3c5c13705820921086da0b08322d526c8967